### PR TITLE
Support for Plone 5 plone.app.contenttypes folder.

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from setuptools import find_packages
 from setuptools import setup
 
-version = '1.11.dev0'
+version = '1.2.dev0'
```
Comment (thet) [inline setup.py]: Please make a 2.0 out of it.
1.2 is smaller than 1.11.

Comment (agitator) [reply setup.py]: done

**Comment by thet** [comment]
@agitator is this really necessary? The plone.app.contenttypes folder should also implement plone.folder.interfaces.IFolder.
I also had never problems using this package under Plone 5.
See:
```
>>> from plone.app.contenttypes.interfaces import IFolder
>>> from plone.folder.interfaces import IFolder
>>> from plone.app.contenttypes.interfaces import IFolder as ctf
>>> from plone.folder.interfaces import IFolder as pff
>>> ctf.providedBy(app.Plone.events)
True
>>> pff.providedBy(app.Plone.events)
True
```

**Comment by agitator** [comment]
weird... seems the extra view registration isn't necessary anymore
can confirm, released version works with 5.1.4

**Pull request closed without merging**